security/pkg/server/ca/server.go

```diff
@@ -152,14 +152,10 @@ func recordCertsExpiry(keyCertBundle *util.KeyCertBundle) {
 	rootCertExpiry, err := keyCertBundle.ExtractRootCertExpiryTimestamp()
```

Comment:
Maybe a comment here as well that this is just the first cert in the trust bundle ? 

I am not entirely sure what is the goal of reporting this - a proper rotation for trust bundles involves adding a new root in the bundle, that expires later. There is no problem if the first root is close to expiration if a new root is available and the cert chain is signed by that root.